src/primo/browzine-primo-adapter.js

```diff
@@ -465,7 +465,6 @@ browzine.primo = (function() {
     showArticleBrowZineWebLinkText: showArticleBrowZineWebLinkText,
     showDirectToPDFLink: showDirectToPDFLink,
     showPrintRecords: showPrintRecords,
-    isFiltered: isFiltered,
```

Comment:
Ooh I see, this was already defined up higher in this object 👍

Reply:
Yep. Code cleanup.